parsl/monitoring/visualization/plots/default/workflow_plots.py

```diff
@@ -197,25 +199,26 @@ def workflow_dag_plot(df_tasks, group_by_apps=True):
             name=k,          # legend app_name here
             marker=dict(
                 showscale=False,
-                # color='rgb(200,0,0)',
+                color=color,
```

Comment:
I would need to try this, and see what happen when there are multiple `apps` and the color is set to `None`

Reply:
The documentation here https://plot.ly/python/reference/#scatter says that colour must be either a colour or a list of colour indices.

Actual behaviour with the version of plotly I'm using is that color=None seems to be the same as not specifying color at all - it allocates a new color automatically.